quick-start/e2e/page-objects/flows/flows.ts

```diff
@@ -212,8 +273,8 @@ export class FlowPage extends AppPage {
 
     jobsPage.isLoaded();
 
-    browser.wait(EC.presenceOf(jobsPage.firstFinishedFlow));
-    expect(jobsPage.finishedFlows.count()).toBe(count);
+    //browser.wait(EC.presenceOf(jobsPage.firstFinishedFlow));
```

Comment:
looks like these two lines can be deleted

Reply:
yes, I plan to change the verification step